components/motor/gpio/motor_encoder.go

```diff
@@ -80,7 +79,6 @@ func newEncodedMotor(
 		ticksPerRotation:  float64(motorConfig.TicksPerRotation),
 		real:              localReal,
 		rampRate:          motorConfig.RampRate,
-		maxPowerPct:       motorConfig.MaxPowerPct,
```

Comment:
I'm having basic handle max power. We call it in both rpm monitor and controls anyway.